src/com/company/Main.java

```diff
@@ -9,19 +10,20 @@ public static void main(String[] args) {
 
         Scanner sc = new Scanner(System.in);
 
-        System.out.println("Do you want to know actual date and time ?" + " Enter Yes/No");
+        System.out.println("Do you want to know actual date and time ? Enter Yes/No");
         String ans = sc.nextLine();
 
+        //inquiry
         if (ans.equals("Yes")) {
```

Comment:
Please, use constants for "yes" and "no". It allows to reuse them in different places and exlude the possibility to make a mistake.

Reply:
Done